subsys/mpsl/fem/Kconfig

```diff
@@ -39,20 +44,37 @@ config MPSL_FEM_PIN_FORWARDER
 
 if MPSL_FEM || MPSL_FEM_PIN_FORWARDER
 
+config MPSL_FEM_USED
```

Comment:
This symbol is redundant. Please use `CONFIG_MPSL_FEM_ANY_SUPPORT` instead in CMakeLists.txt - it should have the same effect without the additional Kconfig option